pkg/k3s/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+
+FROM lfedge/eve-alpine:c5fecbf55e75b10b48a6d4a504d2d9a2e3caf1f9 as build
+ENV PKGS alpine-baselayout musl-utils iproute2 iptables curl openrc \
+	open-iscsi libvirt libvirt-client util-linux grep
+RUN eve-alpine-deploy.sh
+
+FROM scratch
+COPY --from=build /out/ /
+COPY cluster-init.sh /usr/bin/
+COPY cgconfig.conf /etc
+COPY longhorn-config.yaml /etc
+COPY kubevirt-operator.yaml /etc
+WORKDIR /
+
+ENV K3S_VERSION v1.26.3+k3s1
+
+RUN /usr/bin/curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=${K3S_VERSION} INSTALL_K3S_SKIP_ENABLE=true INSTALL_K3S_BIN_DIR=/usr/bin sh -
```

Comment:
You won't to run build with network access, so you should use ADD to fetch from https://get.k3s.io

Reply:
Unfortunately ADD will only help to fetch the script but within the script k3s is downloading few files which require network access. If network access is not allowed at all during build, then I need to think of installing k3s as part of container startup.

Reply:
Does the k3s offline installation requires a lot of changes? If so, I think you can allow network access, we have a few exceptions that does that, but we want to avoid as much as possible....

Reply:
I already updated commit by moving k3s installation to container boot time.